jsign-crypto/src/main/java/net/jsign/jca/AmazonSigningService.java

```diff
@@ -259,7 +274,7 @@ void sign(HttpURLConnection conn, AmazonCredentials credentials, byte[] content,
         String host = endpoint.getHost();
         Matcher matcher = hostnamePattern.matcher(host);
         String regionName = matcher.matches() ? matcher.group(2) : "us-east-1";
-        String serviceName = matcher.matches() ? matcher.group(1) : "kms";
+        String serviceName = "kms";
```

Comment:
I'd rather strip the `-fips` suffix here, so this method remains generic for any AWS service

Reply:
But that class is only intended to access KMS or are there other services involved that I miss?